### PR TITLE
Add LazyLLM and its usage to ecosystem/README.md

### PR DESCRIPTION
**Pull request by JingofXin**
## Motivation

Add LazyLLM and its usage to ecosystem/README.md

## Modification

Add LazyLLM and its usage to ecosystem/README.md

The three examples given in the document have been verified:

1. Chatbot: 
企业微信截图_17254519405423

2. RAG: 
企业微信截图_17254524136833

3. Finetune: 
企业微信截图_17254531019386

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: ecosystem/README_zh-CN.md [hunk 1]
```diff
@@ -244,3 +244,54 @@ LlamaIndex 是一个用于构建上下文增强型 LLM 应用程序的框架。
...
+# will launch a chatbot server
+WebModule(m).start().wait()
+```
+
+如果您需要进一步微调模型，可以参考如下代码。当 `TrainableModule` 的 `trainset` (数据集需下载到本地，例如：[
+alpaca_gpt4_zh](https://huggingface.co/datasets/llamafactory/alpaca_gpt4_zh))被设置之后，在调用 `WebModule` 的 `update` 函数时，会自动微调 `TrainableModule`，然后对 `TrainableModule` 和 `WebModule` 分别进行部署。
+
+```python
+from lazyllm import TrainableModule, WebModule
+m = TrainableModule('internlm2-chat-7b').trainset('/patt/to/your_data.json').mode('finetune')
```
Comment (zhulinJulia24) [inline ecosystem/README_zh-CN.md:266]: Using llamafactory backend to do internlm2.5 sft. 

error is
LesUdwiEHe

It seems llamafactory do not have a good support of internlm2.5

File: ecosystem/README_zh-CN.md [hunk 2]
```diff
@@ -244,3 +244,54 @@ LlamaIndex 是一个用于构建上下文增强型 LLM 应用程序的框架。
...
+import os
+import lazyllm
+from lazyllm import pipeline, parallel, bind, SentenceSplitter, Document, Retriever, Reranker
+
+prompt = '你将扮演一个人工智能问答助手的角色，完成一项对话任务。在这个任务中，你需要根据给定的上下文以及问题，给出你的回答。'
+```
+</details>
+
+```python
+documents = Document(dataset_path='/file/to/yourpath', embed=lazyllm.TrainableModule('bge-large-zh-v1.5'))
```
Comment (zhulinJulia24) [inline ecosystem/README_zh-CN.md:285]: So weird. I download rag_master dataset first under /home/zhulin1/rag_master
```
(testall) [zhulin1@SH-IDC1-10-140-0-187 rag_master]$ ls
README.md  中庸.txt  大学.txt  心经.txt  论语.txt  道德经.txt  阴符经.txt
(testall) [zhulin1@SH-IDC1-10-140-0-187 rag_master]$ pwd
/home/zhulin1/rag_master
```

then run this python script. 
the file under rag_master is changed into
```
(testall) [zhulin1@SH-IDC1-10-140-0-187 sources]$ pwd
/home/zhulin1/rag_master/default/__data/sources
(testall) [zhulin1@SH-IDC1-10-140-0-187 sources]$ ls
README.md  中庸.txt  大学.txt  心经.txt  论语.txt  道德经.txt  阴符经.txt
```
and have error like this:

```
INFO] model link found at /home/zhulin1/.lazyllm/model/bge-large-zh-v1.5
[INFO] model link found at /home/zhulin1/.lazyllm/model/internlm2_5-1_8b-chat
44145: 2024-09-06 13:09:18 lazyllm INFO: (lazyllm.launcher:107) Command: /home/zhulin1/miniconda3/envs/testall/bin/python /home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/components/deploy/relay/server.py --open_port=34633  
44145: 2024-09-06 13:09:21 lazyllm INFO: (lazyllm.launcher:152) 44409: INFO:     Started server process [44409]
44145: 2024-09-06 13:09:21 lazyllm INFO: (lazyllm.launcher:152) 44409: INFO:     Waiting for application startup.
44145: 2024-09-06 13:09:21 lazyllm INFO: (lazyllm.launcher:152) 44409: INFO:     Application startup complete.
44145: 2024-09-06 13:09:21 lazyllm INFO: (lazyllm.launcher:152) 44409: INFO:     Uvicorn running on http://0.0.0.0:34633 (Press CTRL+C to quit)
44145: 2024-09-06 13:09:21 lazyllm INFO: (lazyllm.components.deploy.base:59) Capture startup message: INFO:     Uvicorn running on http://0.0.0.0:34633 (Press CTRL+C to quit)

44145: 2024-09-06 13:09:21 lazyllm INFO: (lazyllm.launcher:107) Command: /home/zhulin1/miniconda3/envs/testall/bin/python /home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/components/deploy/relay/server.py --open_port=34858  
44145: 2024-09-06 13:09:24 lazyllm INFO: (lazyllm.launcher:152) 44613: INFO:     Started server process [44613]
44145: 2024-09-06 13:09:24 lazyllm INFO: (lazyllm.launcher:152) 44613: INFO:     Waiting for application startup.
44145: 2024-09-06 13:09:24 lazyllm INFO: (lazyllm.launcher:152) 44613: INFO:     Application startup complete.
44145: 2024-09-06 13:09:24 lazyllm INFO: (lazyllm.launcher:152) 44613: INFO:     Uvicorn running on http://0.0.0.0:34858 (Press CTRL+C to quit)
44145: 2024-09-06 13:09:24 lazyllm INFO: (lazyllm.components.deploy.base:59) Capture startup message: INFO:     Uvicorn running on http://0.0.0.0:34858 (Press CTRL+C to quit)

44145: 2024-09-06 13:09:24 lazyllm ERROR: (lazyllm.flow.flow:249) An error occored when invoking `<class 'lazyllm.flow.flow._FuncWrap'>(<Function type=_work>)` with input `()` and kw `{}`
44145: 2024-09-06 13:09:24 lazyllm ERROR: (lazyllm.flow.flow:249) An error occored when invoking `<class 'lazyllm.flow.flow.Pipeline'>(<Flow type=Pipeline items=[]>
 └- <Function type=_work>
)` with input `()` and kw `{}`
Traceback (most recent call last):
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/requests/models.py", line 974, in json
    return complexjson.loads(self.text, **kwargs)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhulin1/test_lazyllm_rag.py", line 16, in <module>
    lazyllm.WebModule(ppl, port=23456).start().wait()
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/module/module.py", line 305, in start
    return self._update(mode=['server'], recursive=True)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/tools/webpages/webmodule.py", line 390, in _update
    super(__class__, self)._update(mode=mode, recursive=recursive)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/module/module.py", line 250, in _update
    Parallel.sequential(*deploy_tasks)()
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 204, in __call__
    output = self._run(args[0] if len(args) == 1 else package(args), **kw)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 485, in _run
    r = package(self.invoke(it, inp, **kw) for it, inp in zip(items, inputs))
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/common/common.py", line 85, in __new__
    return super(__class__, cls).__new__(cls, args[0])
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 485, in <genexpr>
    r = package(self.invoke(it, inp, **kw) for it, inp in zip(items, inputs))
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 247, in invoke
    return it(__input, **kw)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 204, in __call__
    output = self._run(args[0] if len(args) == 1 else package(args), **kw)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 324, in _run
    output = self.invoke(it, output, bind_args_source=bind_args_source, **kw)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 245, in invoke
    return it(*__input, **kw) if isinstance(__input, package) else it(**__input, **kw)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/flow/flow.py", line 21, in __call__
    def __call__(self, *args, **kw): return self.f(*args, **kw)
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/tools/rag/web.py", line 272, in _work
    self.demo = self.web_ui.create_ui()
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/tools/rag/web.py", line 94, in create_ui
    del_select_group = gr.Dropdown(self.list_groups(), label="选择分组")
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/lazyllm/tools/rag/web.py", line 56, in list_groups
    return response.json()["data"]
  File "/home/zhulin1/miniconda3/envs/testall/lib/python3.10/site-packages/requests/models.py", line 978, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```


@JingofXin do you have any suggestion about this error?

**Review by zhulinJulia24 — APPROVED** [review]
LGTM

**Review by lvhan028 — APPROVED** [review]

File: ecosystem/README_zh-CN.md (hunk 1 above)
Comment (JingofXin) [reply ecosystem/README_zh-CN.md:266]: Indeed, the version of LLaMA-Factory used in the current release of lazyllm does not support this 2.5 version of the model. But the latest LLaMA-Factory supported.

**Pull request merged**